library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -572,6 +577,10 @@ data class Conversations(
             override fun onConversation(conversation: FfiConversation) {
                 trySend(Conversation.Group(Group(client, conversation)))
             }
+
+            override fun onError(error: FfiSubscribeException) {
+
+            }
```

Comment:
@insipx @rygine how are you all handling the errors on the web? Should I be throwing here? Or just ignoring the error. @alexrisch what would you prefer on converse? If the stream has issues would you like me to throw something you can catch or just silently error?

Reply:
My inclination is to just Log an error and do nothing.

Reply:
Ignoring the error would be the same behavior as previous. For MessageKit we're letting the user decide how to handle the error (this allows user to restart the stream if the server restarts for instance). we haven't really discussed a strategy for web yet. It would be good to have a consistent strategy across the SDKs, though. Probably OK that MessageKit is a little different since that's for bots